modules/postgresql/variables.tf

```
@@ -138,6 +138,22 @@ variable "backup_configuration" {
   }
 }
 
+variable "insights_config" {
+  description = "The insights_config settings subblock for the database setings"
+  type = object({
+    query_insights_enabled  = bool
```

Instead of having this, we should default to `insights_config` being null and set `query_insights_enabled = true` automatically if the block is provided.

---

Thanks for the review, I have patched for the advice.

---

Sorry, I think I might have been unclear. I want the default for this entire variable to be `null` (disable query insights entirely). Like so:

```suggestion
    query_string_length     = number
    record_application_tags = bool
    record_client_address   = bool
  })
  default = null
```